src/risedevtool/src/config_gen/prometheus_gen.rs

```diff
@@ -70,11 +71,31 @@ impl PrometheusGen {
             .map(|node| format!("\"{}:{}\"", node.address, 9644))
             .join(",");
 
+        let now = Local::now().format("%Y%m%d-%H%M%S");
+
+        let mut remote_write = String::from("");
+        if let Ok(x) = std::env::var("PROMETHEUS_REMOTE_WRITE") && x == "true" {
+            remote_write = "
+remote_write:
```

Comment:
I'd prefer to add this in risedev.yml. For example,

```
- use: prometheus
  remote_write: https://aps-workspaces.ap-southeast-1.amazonaws.com/workspaces/ws-f3841dad-6a5c-420f-8f62-8f66487f512a/api/v1/remote_write
  cluster_label: "2022-07-13"
```

I believe it will have better flexibility for benchmarks.